### PR TITLE
Use a while loop instead of depending on systemd (fixes #673).

### LINKED ISSUE #673: etcd/etcdEvents Cluster Creation Errors
EtcdEvents cluster has periodic problems when starting a new cluster.   
Also, it appears that, if the EtcdEvents cluster was failing to be established...it would never resolve the issue with time (or another up.sh).   You have to tear down and restart the cluster.

cluster error (the ec2 machines were shown as ok in the aws console):
```
mikeln-bender:.kraken Mikel_Nelson$ ssh -o StrictHostKeyChecking=no -o UserKnownHostsFile=/dev/null -F /Users/Mikel_Nelson/dev/cloud/k2new/myconfig/.kraken/mikeln-v17/ssh_config etcdEvents-1 etcdctl --ca-file=/etc/etcd/ssl/client-ca.pem --cert-file=/etc/etcd/ssl/client.pem --key-file=/etc/etcd/ssl/client-key.pem --endpoints=https://127.0.0.1:2381 cluster-health
Warning: Permanently added 'ec2-54-201-86-117.us-west-2.compute.amazonaws.com,54.201.86.117' (ECDSA) to the list of known hosts.
cluster may be unhealthy: failed to list members
Error:  client: etcd cluster is unavailable or misconfigured
error #0: client: endpoint https://127.0.0.1:2381 exceeded header timeout
```
vs good cluster:
```
mikeln-bender:.kraken Mikel_Nelson$ ssh -o StrictHostKeyChecking=no -o UserKnownHostsFile=/dev/null -F /Users/Mikel_Nelson/dev/cloud/k2new/myconfig/.kraken/mikeln-v17/ssh_config etcdEvents-1 etcdctl --ca-file=/etc/etcd/ssl/client-ca.pem --cert-file=/etc/etcd/ssl/client.pem --key-file=/etc/etcd/ssl/client-key.pem --endpoints=https://127.0.0.1:2381 cluster-health
Warning: Permanently added 'ec2-54-202-34-222.us-west-2.compute.amazonaws.com,54.202.34.222' (ECDSA) to the list of known hosts.
member 41a3e112a65a837c is healthy: got healthy result from https://10.0.78.85:2381
member a2c4d959cd59f4f6 is healthy: got healthy result from https://10.0.63.5:2381
member b7f63ca0829db36f is healthy: got healthy result from https://10.0.234.11:2381
member cd552ef50bda685e is healthy: got healthy result from https://10.0.84.207:2381
member dddc44a21e23857e is healthy: got healthy result from https://10.0.201.175:2381
cluster is healthy
```

Also, maybe related.  The `etcdctl` health checks are only run off the first node in the cluster (e.g. etcd-1 and etcdEvents-1).  This might contribute to this issue, if node 1 is having issues, and not restarted automatically. (No evidence of this yet, but a thought).

### PR DESCRIPTION
**Pull request by davidewatson**

### CODE REVIEW

**Comment by leahnp** [comment]
LGTM - do you mind giving me a brief overview of the pros/cons of this approach vs our old systemd one?

File: ansible/roles/kraken.config/files/config.yaml [hunk 1]
```diff
@@ -460,7 +460,7 @@ deployment:
           # 5 nodes recommended for production/high availability clusters. This
           # configuration is required to tolerate a single node failure during
           # node upgrades.
-          count: 1
+          count: 5
```
Comment (coffeepac) [inline ansible/roles/kraken.config/files/config.yaml]: oh?

**Comment by davidewatson** [comment]
@leahnp:

tl;dr; the bash while loop is simpler (and more reliable) because you do not need to understand as much about systemd.

"Restart=on-failure only applies to process failures (does not apply to failure due to dependency failures)", and "If there's even a small chance that a dependency might fail on start and you care about resiliency, don't use Before/After and instead perform a check on some artifact that the dependency produces" - https://unix.stackexchange.com/a/216254

But, having to create an artifact is more trouble than we want than we'd like if we can avoid it.
